Faceboxes模型导出失败

python3 tools/export_model.py -c configs/face_detection/faceboxes_lite.yml --output_dir=./inference_model   -o weights=https://paddlemodels.bj.bcebos.com/object_detection/faceboxes_lite.tar TestReader.inputs_def.image_shape=[3,640,640]

执行上面的命令，出现下面的错误


AssertionError: Parameter file [~/.cache/paddle/weights/faceboxes_lite.pdparams] not exits

--- a/slim/distillation/distill.py
+++ b/slim/distillation/distill.py
@@ -32,7 +32,7 @@ from ppdet.data.reader import create_reader
 from ppdet.utils.eval_utils import parse_fetches, eval_results, eval_run
 from ppdet.utils.stats import TrainingStats
 from ppdet.utils.cli import ArgsParser
-from ppdet.utils.check import check_gpu, check_config
+from ppdet.utils.check import check_gpu, check_version, check_config
 import ppdet.utils.checkpoint as checkpoint
 
 import logging
@@ -134,6 +134,7 @@ def main():
     check_config(cfg)
     # check if set use_gpu=True in paddlepaddle cpu version
     check_gpu(cfg.use_gpu)
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/slim/extensions/distill_pruned_model/distill_pruned_model.py
+++ b/slim/extensions/distill_pruned_model/distill_pruned_model.py
@@ -35,7 +35,7 @@ from ppdet.data.reader import create_reader
 from ppdet.utils.eval_utils import parse_fetches, eval_results, eval_run
 from ppdet.utils.stats import TrainingStats
 from ppdet.utils.cli import ArgsParser
-from ppdet.utils.check import check_gpu, check_config
+from ppdet.utils.check import check_gpu, check_version, check_config
 import ppdet.utils.checkpoint as checkpoint
 
 import logging
@@ -123,6 +123,7 @@ def main():
     check_config(cfg)
     # check if set use_gpu=True in paddlepaddle cpu version
     check_gpu(cfg.use_gpu)
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/slim/prune/export_model.py
+++ b/slim/prune/export_model.py
@@ -27,7 +27,7 @@ from paddle import fluid
 from ppdet.core.workspace import load_config, merge_config, create
 from ppdet.utils.cli import ArgsParser
 import ppdet.utils.checkpoint as checkpoint
-from ppdet.utils.check import check_config
+from ppdet.utils.check import check_config, check_version
 from paddleslim.prune import Pruner
 from paddleslim.analysis import flops
 
@@ -82,6 +82,7 @@ def main():
     cfg = load_config(FLAGS.config)
     merge_config(FLAGS.opt)
     check_config(cfg)
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/slim/quantization/export_model.py
+++ b/slim/quantization/export_model.py
@@ -27,7 +27,7 @@ from paddle import fluid
 from ppdet.core.workspace import load_config, merge_config, create
 from ppdet.utils.cli import ArgsParser
 import ppdet.utils.checkpoint as checkpoint
-from ppdet.utils.check import check_config
+from ppdet.utils.check import check_config, check_version
 from tools.export_model import prune_feed_vars
 
 import logging
@@ -57,6 +57,7 @@ def main():
     cfg = load_config(FLAGS.config)
     merge_config(FLAGS.opt)
     check_config(cfg)
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/slim/sensitive/sensitive.py
+++ b/slim/sensitive/sensitive.py
@@ -53,6 +53,7 @@ def main():
     cfg = load_config(FLAGS.config)
     merge_config(FLAGS.opt)
     check_config(cfg)
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/tools/export_model.py
+++ b/tools/export_model.py
@@ -28,7 +28,7 @@ from paddle import fluid
 from ppdet.core.workspace import load_config, merge_config, create
 from ppdet.utils.cli import ArgsParser
 import ppdet.utils.checkpoint as checkpoint
-from ppdet.utils.check import check_config
+from ppdet.utils.check import check_config, check_version
 import yaml
 import logging
 from collections import OrderedDict
@@ -175,6 +175,8 @@ def main():
     cfg = load_config(FLAGS.config)
     merge_config(FLAGS.opt)
     check_config(cfg)
+
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/tools/export_serving_model.py
+++ b/tools/export_serving_model.py
@@ -22,7 +22,7 @@ from paddle import fluid
 
 from ppdet.core.workspace import load_config, merge_config, create
 from ppdet.utils.cli import ArgsParser
-from ppdet.utils.check import check_config
+from ppdet.utils.check import check_config, check_version
 import ppdet.utils.checkpoint as checkpoint
 import yaml
 import logging
@@ -58,6 +58,7 @@ def main():
     cfg = load_config(FLAGS.config)
     merge_config(FLAGS.opt)
     check_config(cfg)
+    check_version()
 
     main_arch = cfg.architecture
 

--- a/tools/face_eval.py
+++ b/tools/face_eval.py
@@ -30,7 +30,7 @@ from collections import OrderedDict
 
 import ppdet.utils.checkpoint as checkpoint
 from ppdet.utils.cli import ArgsParser
-from ppdet.utils.check import check_gpu, check_config
+from ppdet.utils.check import check_gpu, check_version, check_config
 from ppdet.utils.widerface_eval_utils import get_shrink, bbox_vote, \
     save_widerface_bboxes, save_fddb_bboxes, to_chw_bgr
 from ppdet.core.workspace import load_config, merge_config, create
@@ -219,6 +219,7 @@ def main():
     check_config(cfg)
     # check if set use_gpu=True in paddlepaddle cpu version
     check_gpu(cfg.use_gpu)
+    check_version()
 
     main_arch = cfg.architecture
 

